elastic/kibana
# [Rollup Jobs] Added autofocus to cron editor

## Summary
Fixes [#28298](https://github.com/elastic/kibana/issues/28298).

This PR adds an autofocus when the cron editor is switched between a set of selects for frequency to an input text field. This removes a confusing step of tabbing backwards when using a keyboard when navigating the form. 

## Gif
![cron](https://user-images.githubusercontent.com/6585477/102528590-0f8feb80-409f-11eb-93ad-86a819d2d422.gif)

## How to test
Navigate to Rollup Jobs and use keyboard in the new job creation form. When editing frequency, switch between input field and a set of selects and make sure the input field or the first select is auto focused. Navigate between the wizard steps to make sure, the cron editor is not auto focused on component load. 

## Release Note 
To make navigating the rollup job form with a keyboard easier, it now has an autofocus on the input field after the switch between cron expression and frequency selection.
